src/Service/StatsLoggingService.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace App\Service;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpKernel\Event\TerminateEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Class StatsLoggingService.
+ */
+class StatsLoggingService implements LoggerInterface
+{
+    const EMERGENCY = 'emergency';
```

Comment:
Is these not defined in the `LoggerInterface`? or can be extended from some where?

Reply:
Fixed